PWGLF/Tasks/Nuspex/LFNucleiBATask.cxx

```diff
@@ -71,7 +71,7 @@ struct LFNucleiBATask {
   Configurable<bool> enableAl{"enableAl", true, "Flag to enable alpha analysis."};
 
   Configurable<bool> enableTrackingEff{"enableTrackingEff", 0, "Flag to enable tracking efficiency hitos."};
-  Configurable<std::string> ccdburl{"ccdb-url", "http://alice-ccdb.cern.ch", "url of the ccdb repository"};
+  Configurable<std::string> ccdburl{"ccdb-url", "http://alice-ccdb.cern.ch", "url of the ccdb repository"}; // NOLINT
```

Comment:
Why?
- This will not work.
- Silencing is not well justified here.

Reply:
@vkucera  I have seen that ccdburl variable is usually written all lowercase in most of the analyses and I don't want to change it